README.md

```diff
@@ -1139,6 +1139,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Shivam Katare](https://github.com/Shivam-Katare)
   - [Shivendra K Jha](https://github.com/skjha1)
   - [Shivendu Kumar](https://github.com/IAMSKJha)
+  - [Shivam Sharma](https://github.com/shivam-sharma7/)
```

Comment:
you should remove "/" in the end

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix